src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/ColorFieldTransformer.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import React from 'react';
+import type {Node} from 'react';
+import type {FieldTransformer} from '../types';
+
+export default class ColorFieldTransformer implements FieldTransformer {
+    transform(value: *): Node {
+        return <div style={{'backgroundColor': value, 'width': '40px', 'height': '20px'}}></div>;
```

Comment:
Would add a `className` and create a `.scss` which is imported, which set the `width` and `height` attribute.

Reply:
/cc @chirimoya any preferences how the color should be displayed?

Reply:
My first commit was to fast and to light :)
Here come a new commit with your requirements and more control in the transformer.
First time i handle all this tools : was very cool 👍